source/installation/with/snap.html.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Installing from the Snap store
+---
+
+Plume is available on the [Snap Store](https://snapcraft.io/plume). The snap package is
+isolated from your host and will automatically update, running any necessary migration
+tasks.
```

Comment:
one sentence per line, please
also, an example of how to install from command line would be appreciated

Reply:
How's this?